.travis.yml

```diff
@@ -30,6 +30,43 @@ script:
     # traces from any crashes.
     - export LD_PRELOAD=/lib/x86_64-linux-gnu/libSegFault.so
     - export LD_LIBRARY_PATH=$DEPENDENCIES/lib
+    - export IECORE_STREAMINDEXEDIO_COMPRESSION="lz4 0 1 1"
+    - scons -j 2
```

Comment:
I suspect this is going to be a real pain to maintain, remembering to update _both_ scons commands each time. I'm sure I'll forget. It'll also make the logs a little bit harder to follow...

Did you consider instead adding the extra permutation into the build matrix at the bottom of this file? The downside would be that we'd be paying for another build, but the upside would be easy maintenance of the config, and very clear reporting of the results as a separate job.

Reply:
> Did you consider instead adding the extra permutation into the build matrix at the bottom of this file? The downside would be that we'd be paying for another build, but the upside would be easy maintenance of the config, and very clear reporting of the results as a separate job.

This did not occur to me. Sounds much better.